css/main.css

```diff
@@ -143,92 +143,95 @@ h2#Sverde{
 
 /* -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales*/
 
-div#divServicios ul li{
+div#div-servicios ul li {
     margin-bottom: 10px;
 }
 
-div#divServicios #serviceCon{
+div#div-servicios #service-con {
     display: flex;
     flex-wrap: wrap;
     padding: 180px;
 }
 
-div.service{
+div.service {
     width: 175px;
     display: inline-block;
     margin-left: 50px;
     margin: 10px;
     text-align: center;
-    flex:1;
+    flex: 1;
 }
 
-div#divServicios p{
+div#div-servicios p {
     text-transform: uppercase;
     padding-bottom: 15px;
 }
 
 /* -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales*/
 
-div#divClientes{   
+div#div-clientes {   
     background-color: #24BDC5;
 }
 
 /* -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales*/
 
-div#divProductos{
+div#div-productos {
     display: flex;
     align-items: center;
     flex-direction: column;
     text-align: center;
     background-color: rgb(255, 255, 255);   
```

Comment:
Let's use the hex convention please to keep an standard

Reply:
done